docs/application/web/guides/multimedia/metadata.md

```diff
@@ -0,0 +1,43 @@
+# Metadata
+
+You can extract metadata information from multimedia files.
+
+The main features of the Metadata API include:
+
+- Extraction of a metadata from a file
+  You can [extract metadata](#extract-metadata) from multimedia files using `get()` method.
+
+
+## Prerequisites
+
+To access files using the Metadata API (in [mobile](../../api/latest/device_api/mobile/tizen/metadata.html), [wearable](../../api/latest/device_api/wearable/tizen/metadata.html)), the application has to request [proper storage permissions](../security/privacy-related-permissions.md).
+
+## Extract metadata
```

Comment:
The heading follows the latest prescribed sentence style.
@mijin85cho Should we change it to Title case to maintain consistency as conversion task will follow up to change the cases. 

## Extract Metadata

Reply:
As this page is newly added one, let's use the new style that will be applied to whole pages.
I suggest to keep the current sentence style.